src/eth/execute.yaml

```
@@ -54,3 +54,14 @@
         gasUsed:
           title: Gas used
           $ref: '#/components/schemas/uint'
+- name: eth_batchCall
```

```suggestion
- name: eth_multicall
```
I implemented this feature (more or less) in Nethermind as a plugin a long time ago and I called it `eth_multicall` so I will admit I'm a bit biased here, but I like the name `multicall` better than `batchCall`.  Especially since there is already JSON-RPC batching that may be confused with this feature.

---

Also, batching to me implies "shipped at the same time" but not necessarily "processed in order as a sequential series of events".  I'm not sure `multicall` implies this clearly, but I think batch implies the opposite.

---

I also like the `eth_multicall` name.

---

> I also like the `eth_multicall` name.

you may find this interesting then for the erigion implementation of `eth_multiCall` 
https://github.com/manifoldfinance/golang-eth-patches/blob/master/EthMultiCall_RPC.go.patch